packages/browser-wallet/src/background/index.ts

```diff
@@ -430,14 +417,14 @@ bgMessageHandler.handleMessage(
 );
 
 const getSelectedChainHandler: ExtensionMessageHandler = (_msg, sender, respond) => {
-    isWalletLocked().then((locked) => {
-        if (locked) {
+    if (!sender.url) {
+        throw new Error('Expected URL to be available for sender.');
+    }
+
+    isAllowListedForAnyAccount(sender.url).then((allowListed) => {
```

Comment:
There was no prior restriction for being allowlisted for this before?

Reply:
Nope, before it just required unlocking. :thinking:  
I'll remove the restriction.